segmentation/utils/data_utils.py

```diff
@@ -233,16 +241,17 @@ def load_imgs_from_dir(data_dir, image_name='img_data', delimiter='_',
     fovs = [img.split(delimiter)[0] for img in imgs]
 
     img_xr = xr.DataArray(img_data, coords=[fovs, row_coords, col_coords, [image_name]],
-                          dims=["fovs", "rows", "cols", "channels"])
+                          dims=["fovs", "rows", "cols", "compartments"])
```

Comment:
Do we use this function for anything else? Changing the dim name is fine if it's only used for loading segmentation masks

Reply:
Actually, can have both the coordinate name and the dimension name be parameters of the function?

Reply:
I'm pretty sure this is just for loading segmentation masks.  The dim used to be called 'channels' but it looks like somebody updated that at some point, which went unnoticed since I forgot to actually try using `tiffs_to_xr_labels` in the nb's or test its use in plotting functions, etc. within a test function.

Reply:
Regardless, I can add the option to also name the dim in the `load_imgs_from_dir` function